MANIFEST.in

```
@@ -1 +1,15 @@
-prune tests/data
+include README.md
+include LICENSE
+include pyproject.toml
+
+exclude *.yml
+exclude *.yaml
+exclude tox.ini
+exclude CHANGELOG.md
+
+graft src
+
+prune benchmarks
+prune tests
+
+exclude cellfinder-core/benchmarks/*
```

```suggestion
prune cellfinder-core/benchmarks/
```

`exclude`-ing a file pattern will still ship an empty folder, whereas `prune` will completely get rid of it.

Also there are some conflicts between the version of this file on `main` and on this branch, but it looks like it's just `git` being confused by the addition of a couple of lines :)